web_client/src/components/Navbar.vue

```diff
@@ -72,7 +72,11 @@ export default defineComponent({
         <template #activator="{ on }">
           <span v-on="on">MIQA</span>
         </template>
-        <span>{{ MIQAConfig.version.length > 0 ? MIQAConfig.version : "Demo Instance" }}</span>
+        <span>{{
+          MIQAConfig.version && MIQAConfig.version.length > 0
```

Comment:
Does this come from the server? Can we just build it into the client so that it's guaranteed to be string-valued?